src/ucp/core/ucp_ep.c

```diff
@@ -1468,6 +1465,22 @@ ucs_status_t ucp_ep_config_init(ucp_worker_h worker, ucp_ep_config_t *config,
     get_zcopy_lane_count = 0;
     put_zcopy_lane_count = 0;
     rndv_max_bw          = 0;
+    for (i = 0; (i < config->key.num_lanes) &&
+                (config->key.rma_bw_lanes[i] != UCP_NULL_LANE); ++i) {
+        lane      = config->key.rma_bw_lanes[i];
+        rsc_index = config->key.lanes[lane].rsc_index;
+
+        if (rsc_index != UCP_NULL_RESOURCE) {
```

Comment:
why would rsc_index be NULL? should we assert here?

Reply:
it can be `UCP_NULL_RESOURCE` in case of
```
ucp_worker_iface_err_handle_progress()
```
where we set `lanes[0]` to `UCP_NULL_RESOURCE`

Reply:
so 
if (rsc_index == NULL_RESOURCE) 
 continue

Reply:
done